### ISSUE TITLE
findfont: Font family ['Helvetica'] not found. Falling back to DejaVu Sans.

### ISSUE THREAD

**Issue by luispsa** [dexplo/bar_chart_race#32]
Hi, I get this issue when I load the example script

findfont: Font family ['Helvetica'] not found. Falling back to DejaVu Sans.
findfont: Font family ['Helvetica'] not found. Falling back to DejaVu Sans.
findfont: Font family ['Helvetica'] not found. Falling back to DejaVu Sans.
D:\ProgramData\Anaconda3\lib\site-packages\bar_chart_race\_make_chart.py:286: UserWarning: FixedFormatter should only be used together with FixedLocator
  ax.set_yticklabels(self.df_values.columns)
D:\ProgramData\Anaconda3\lib\site-packages\bar_chart_race\_make_chart.py:287: UserWarning: FixedFormatter should only be used together with FixedLocator
  ax.set_xticklabels([max_val] * len(ax.get_xticks()))

I'm using python 3.8.5 and matplotlib 3.3.1

**Comment by ahmetakil** [comment]
I also get the same error but it doesn't seem to effect anything

**Comment by luispsa** [comment]
> I also get the same error but it doesn't seem to effect anything

yep, the I fixed the FixedFormatter bug  downgrading matplotlib to 3.2.2 version
